go/engine/identify2_with_uid.go

```diff
@@ -384,6 +397,7 @@ func (e *Identify2WithUID) loadMe(ctx *Context) (err error) {
 func (e *Identify2WithUID) loadThem(ctx *Context) (err error) {
 	arg := libkb.NewLoadUserArg(e.G())
 	arg.UID = e.arg.Uid
+	arg.ResolveBody = e.ResolveBody
```

Comment:
We can save an RT to the server if we plumb the other fields from resolve through to the identification call.